src/bentoml/_internal/frameworks/transformers.py

```diff
@@ -605,6 +643,337 @@ def make_default_signatures(pretrained: t.Any) -> ModelSignaturesType:
     return {k: default_config for k in infer_fn}
 
 
+def import_model(
+    name: Tag | str,
+    model_name_or_path: str | os.PathLike[str],
+    *,
+    task_name: str | None = None,
+    task_definition: dict[str, t.Any] | TaskDefinition | None = None,
+    proxies: dict[str, str] | None = None,
+    revision: str = "main",
+    variant: str | None = None,
```

Comment:
You should handle huggingface_hub kwargs separately? This I assume is the kwargs for passing into huggingface_hub?